cashctrl_ledger/ledger.py

```diff
@@ -45,13 +45,10 @@ def dump_to_zip(self, archive_path: str):
                     default_settings[key] = self._client.account_from_id(system_settings[key])
             settings["DEFAULT_SETTINGS"] = default_settings
 
-            archive.writestr('settings.json', json.dumps(settings))
-            archive.writestr('ledger.csv', self.ledger().to_csv(index=False))
-            archive.writestr('tax_codes.csv', self.tax_codes().to_csv(index=False))
-            archive.writestr('accounts.csv', self.accounts().to_csv(index=False))
+            archive.writestr('settings.yml', json.dumps(settings))
 
     def restore_from_zip(self, archive_path: str):
```

Comment:
Why do we need a `restore_from_zip` method in CashCtrlLedger?

The `LedgerEngine. restore_from_zip()`method should also work CashCtrlLedger.